<ui version="4.0" >
 <author>Martin Pahl</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Geo Data Analyzer - GeoAIDA</string>
  </property>
  <property name="dockOptions" >
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QDockWidget" name="dockWidget" >
      <property name="floating" >
       <bool>false</bool>
      </property>
      <property name="features" >
       <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable|QDockWidget::NoDockWidgetFeatures</set>
      </property>
      <property name="windowTitle" >
       <string>Semantic Net Editor</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents" >
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QPushButton" name="pushButton" >
            <property name="text" >
             <string>Expand Tree</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2" >
            <property name="text" >
             <string>Collapse Tree</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>531</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTreeView" name="semNetEditor_" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="MinimumExpanding" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="AttributeEditor" name="attributeEditor_" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="MinimumExpanding" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="itemsExpandable" >
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDockWidget" name="taskViewerDock_" >
      <property name="floating" >
       <bool>false</bool>
      </property>
      <property name="features" >
       <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable|QDockWidget::NoDockWidgetFeatures</set>
      </property>
      <property name="windowTitle" >
       <string>Taskviewer</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_2" >
       <layout class="QVBoxLayout" >
        <item>
         <widget class="TaskViewer" name="taskViewer_" >
          <property name="columnCount" >
           <number>3</number>
          </property>
          <column>
           <property name="text" >
            <string>PID</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Node</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Command</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="action_New" />
    <addaction name="actionNetLoad" />
    <addaction name="action_Save_Net" />
    <addaction name="actionLoad_Result" />
    <addaction name="actionSave_Result" />
    <addaction name="actionProjectLoad" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menu_View" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <addaction name="action_Scene_Viewer" />
    <addaction name="action_Trash_Viewer" />
    <addaction name="action_Map_Viewer" />
    <addaction name="actionDebug_Labelimage_Viewer" />
   </widget>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About" />
   </widget>
   <widget class="QMenu" name="menu_Analyze" >
    <property name="title" >
     <string>&amp;Analyze</string>
    </property>
    <addaction name="actionStart" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menu_View" />
   <addaction name="menu_Help" />
   <addaction name="menu_Analyze" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="action_New" />
   <addaction name="actionNetLoad" />
   <addaction name="action_Save_Net" />
   <addaction name="separator" />
   <addaction name="separator" />
   <addaction name="actionStart" />
  </widget>
  <action name="action_New" >
   <property name="icon" >
    <iconset resource="gda.qrc" >:/gda/filenew.xpm</iconset>
   </property>
   <property name="text" >
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionNetLoad" >
   <property name="icon" >
    <iconset resource="gda.qrc" >:/gda/fileopen.xpm</iconset>
   </property>
   <property name="text" >
    <string>&amp;Load Net</string>
   </property>
  </action>
  <action name="action_Save_Net" >
   <property name="icon" >
    <iconset resource="gda.qrc" >:/gda/filesave.xpm</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save Net</string>
   </property>
  </action>
  <action name="actionLoad_Result" >
   <property name="text" >
    <string>Load Result</string>
   </property>
  </action>
  <action name="actionSave_Result" >
   <property name="text" >
    <string>Save Result</string>
   </property>
  </action>
  <action name="actionProjectLoad" >
   <property name="text" >
    <string>Load &amp;Project</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
   <property name="shortcutContext" >
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="action_Scene_Viewer" >
   <property name="text" >
    <string>&amp;Scene Viewer</string>
   </property>
  </action>
  <action name="action_Trash_Viewer" >
   <property name="text" >
    <string>&amp;Trash Viewer</string>
   </property>
  </action>
  <action name="action_Map_Viewer" >
   <property name="text" >
    <string>&amp;Map Viewer</string>
   </property>
  </action>
  <action name="actionDebug_Label_Image_Viewer" >
   <property name="text" >
    <string>Debug Label Image Viewer</string>
   </property>
  </action>
  <action name="actionDebug_Labelimage_Viewer" >
   <property name="text" >
    <string>Debug Labelimage Viewer</string>
   </property>
  </action>
  <action name="action_About" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Analyze" >
   <property name="text" >
    <string>&amp;Analyze</string>
   </property>
  </action>
  <action name="actionStart" >
   <property name="icon" >
    <iconset resource="gda.qrc" >:/gda/start.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Start</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AttributeEditor</class>
   <extends>QTreeView</extends>
   <header>attributeeditor.h</header>
  </customwidget>
  <customwidget>
   <class>TaskViewer</class>
   <extends>QTreeWidget</extends>
   <header>TaskViewer</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="gda.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>semNetEditor_</receiver>
   <slot>expandAll()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>102</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel" >
     <x>110</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>semNetEditor_</receiver>
   <slot>collapseAll()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>545</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel" >
     <x>237</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX" >
   <number>10</number>
  </property>
  <property name="gridDeltaY" >
   <number>10</number>
  </property>
  <property name="gridSnapX" >
   <bool>true</bool>
  </property>
  <property name="gridSnapY" >
   <bool>true</bool>
  </property>
  <property name="gridVisible" >
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
